storybook/stories/API/cartesian/Scatter.stories.tsx

```diff
@@ -465,49 +465,6 @@ const ReactiveProps = {
   },
 };
 
-const OtherProps = {
-  xAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  yAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  zAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  top: {
-    table: {
-      category: 'Other',
-    },
-  },
-  left: {
-    table: {
-      category: 'Other',
-    },
-  },
-};
-
-const InternalProps = {
-  points: {
-    description: 'The coordinates of all the scatters.',
-    table: {
-      category: 'Internal',
-    },
-  },
-  activeIndex: {
```

Comment:
will also need a way to perform the same controls as with active index (defaultIndex?)

Reply:
The activeIndex is coming through tooltipContext, I suspect I should have removed it from the docs long ago. `defaultIndex` should propagate to the same property.

Reply:
defaultIndex as it exists is not the same as activeIndex. So it will either need to become like it, or we'll need another property on tooltip?

Reply:
What I'm trying to say is that this prop has not been used on Scatter for a while, and is following whatever index is active in Tooltip.

Reply:
sounds good